### ISSUE TITLE
OpenComputers locks up on chunk loading or moving ship

### ISSUE THREAD

**Issue by dustinsurwill** [LemADEC/WarpDrive#420]
**Describe the bug**
Any opencomputers computer that is on when warping will freeze. The only way to use the computer again is to reboot it.

**To Reproduce**
1. make small ship to test
2. place opencomputers computer and install openos
3. install warpshipcore
4. reboot
5. warp
6. try to use computer after warp

**Expected behavior**
Either the opencomputers computer rebooting or not freezing (still able to be used) after warp

**Logs**
https://gist.github.com/falconraptor/e6b1d7cc686eb6fcf5f00f54d564d261

**Versions** (latest is not a valid answer)
 - WarpDrive version: 1.5.11
 - OpenComputers version: 1.7.5.192
 - Forge version: forge-14.23.5.2847
 - Modpack name/version:
          testpack ver 2.8.5 -> https://www.curseforge.com/minecraft/modpacks/techraptor
          also available to download (exact version instead of week or 2 old) from http://techraptor.us/Test.zip
 - Other notable mods & version: cctweaked: 1.85.2

**Comment by LemADEC** [comment]
This is a known issue in one of OpenComputers dependencies, here: https://github.com/fnuecke/eris/issues/29

WarpDrive LUA scripts are affected by this, see https://github.com/LemADEC/WarpDrive/issues/265 .

**Comment by LemADEC** [comment]
For the record, issue is also reproduced in the LUA console, without WarpDrive.

**Comment by IpsumCapra** [comment]
I have made a custom OS for OpenComputers that bypasses this problem. It can be found [here](https://github.com/IpsumCapra/OpenComputersScripts/tree/master/WarpOS). I'd love to see feedback, feel free to ask me questions if you run into any problems with it.

**Comment by kappa-maintainer** [comment]
@IpsumCapra Good job, the OS works perfectly.

**Comment by Furrure** [comment]
Also, you don't need to reboot the computer, replacing the display and causing a screen update will do the trick

**Comment by rscrappy** [comment]
Hello, it appears that this issue is back, or never fixed. Upon testing in both singleplayer, and on a server, the system still freezes after jumping. I am using OpenComputers 1.7.5.192, With WarpDrive 1.5.19. I have tried a few fixes, none have worked.

**Comment by LemADEC** [comment]
the issue was never fixed in the first place, hence why the ticket is still open...

**Comment by rscrappy** [comment]
Oh, of course, my apologies, I thought there was a patch. I should have paid attention.

**Comment by kappa-maintainer** [comment]
> Oh, of course, my apologies, I thought there was a patch. I should have paid attention.

You can try @IpsumCapra 's script, it works well without rebooting in my game.

**Comment by rscrappy** [comment]
> > Oh, of course, my apologies, I thought there was a patch. I should have paid attention.
> 
> You can try @IpsumCapra 's script, it works well without rebooting in my game.

Yeah, I did try that, but for some reason jumping wouldn't work. I was able to set a target and such, but it simply wouldn't jump.

**Comment by kappa-maintainer** [comment]
The script works well in my game, maybe something got wrong in the installation process ?

**Comment by CSdiscoveries** [comment]
same problem

**Comment by Jmooroof** [comment]
no official fix yet? geez, i've been waiting four years
